app/src/main/java/org/simple/clinic/bloodsugar/entry/BloodSugarEntrySheet.kt

```diff
@@ -383,6 +383,16 @@ class BloodSugarEntrySheet : BottomSheetActivity(), BloodSugarEntryUi, RemoveBlo
     bloodSugarReadingUnitButton.text = getString(R.string.bloodsugarentry_mg_dl)
   }
 
+  override fun showBloodSugarUnitPreferenceButton() {
+    bloodSugarReadingUnitButton.visibility = View.VISIBLE
+    bloodSugarReadingUnitLabel.visibility = View.GONE
```

Comment:
Can we separate hiding the `bloodSugarReadingUnitLabel` visibility to it's own method please?